ibl_to_nwb/datainterfaces/iblsortingextractor.py

```diff
@@ -80,9 +83,15 @@ def __init__(self, session: str, cache_folder: Optional[DirectoryPath] = None):
                 spike_count="spike_count",
                 firing_rate="firing_rate",
                 label="label",
+                uuid="uuid",
+                cluster_id="cluster_id",
```

Comment:
For both of these two sorting properties, I'd recommend including new entries for them in the metadata here: https://github.com/int-brain-lab/IBL-to-nwb/blob/main/ibl_to_nwb/metadata/ecephys.yml#L83-L84

(with useful descriptions paired with their names as defined here)